includes/Core/Modules/Modules.php

```diff
@@ -379,6 +379,15 @@ function ( $data ) {
 			}
 		);
 
+		add_filter(
+			'googlesitekit_module_exists',
+			function ( $exists, $slug ) {
+				return $this->module_exists( $slug );
+			},
+			10,
+			2
+		);
```

Comment:
Please add coverage for this in `ModulesTest::test_register`.

Reply:
Good shout, I've added some test coverage :+1: